src/mender-update/progress_reader/progress_reader.cpp

```diff
@@ -30,6 +30,7 @@ expected::ExpectedSize Reader::Read(
 		int percentage = (bytes_read_ / static_cast<float>(tot_size_)) * 100;
 		if (percentage > last_percentage) {
 			cerr << "\r" << percentage << "%";
+			last_percentage = percentage;
```

Comment:
Minor & preexisting: `last_percentage` should be called `last_percentage_`.

Reply:
I am fixing but I won't run a new integration pipeline for it.